ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/JTATransactionLogXAResourceOrphanFilter.java

```diff
@@ -127,6 +127,7 @@ private boolean transactionLog(Xid xid)
             catch (Exception ex)
             {
                 ex.printStackTrace();
+                return true;
```

Comment:
It is a shame that we cannot return one of "yes/no/maybe" - why did you discount letting the exception leak out of the method for the caller to handle?